scripts/apps/users/directives/UserEditDirective.js

```diff
@@ -26,6 +26,10 @@ export function UserEditDirective(api, gettext, notify, usersService, userList,
             scope.dirty = false;
             scope.errorMessage = null;
 
+            api.query('client_config').then((result) => {
```

Comment:
you should use [deployConfig.get](https://github.com/superdesk/superdesk-client-core/blob/master/scripts/core/config.js#L43) so it can fetch the config just once